utilities/pipelines/dependencies/Microsoft.Storage/storageAccounts/parameters/synapse01.parameters.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
I'd suggest to set the minimum required parameters for the dependencies (e.g. I guess roleAssignments on the blob are not required). This is valid also for synapse02.

Reply:
alright, will remove them. To be honest though - I don't know what requirements are required. This was originally implmented by @renepajta